packages/bundle-size/src/utils/buildFixture.js

```diff
@@ -83,10 +84,12 @@ function webpackAsync(webpackConfig) {
 module.exports = async function buildFixture(preparedFixture, quiet) {
   const webpackStartTime = process.hrtime();
 
+  const config = await readConfig();
```

Comment:
```suggestion
  const config = await readConfig(quiet);
```

Can we pass down `quiet` to have consistent logging with commands?

Reply:
yeah sounds good